focal/Dockerfile

```diff
@@ -134,6 +134,16 @@ RUN apt-get update \
  && cd / \
  && rm -rf /tmp/*
 
+# Protoc and elixir-protobuf plugin
+# Installing protoc from a binary release to avoid ancient packages of ubuntu
+RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v24.2/protoc-24.2-linux-x86_64.zip -O protoc.zip \
+  && mkdir -p ~/.local/protoc/ \
+  && unzip protoc.zip -d $HOME/.local/protoc \
+  && rm protoc.zip \
+  && echo "export PATH='$PATH:$HOME/.local/protoc/bin'" >> $HOME/.bashrc \
+  && mix escript.install hex protobuf --force \
+  && asdf reshim
```

Comment:
why is it needed?

Reply:
From https://github.com/elixir-protobuf/protobuf:

"Make sure the protoc-gen-elixir binary is in your PATH. Either add PATH=~/.mix/escripts:$PATH to your bash or zsh profile or, if you used asdf to install elixir, run asdf reshim and then verify that protoc-gen-elixir works"